paperqa/docs.py

```diff
@@ -525,12 +527,12 @@ async def adoc_match(
         if len(matched_docs) == 0:
             return set()
         # this only works for gpt-4 (in my testing)
-        try:
+        with contextlib.suppress(AttributeError):
             if (
                 rerank is None
                 and (
                     isinstance(self.llm_model, OpenAILLMModel)
-                    and self.config["model"].startswith("gpt-4")
+                    and self.llm_model.config["model"].startswith("gpt-4")
```

Comment:
@whitead FYI previously this always hit an `AttributeError`, so we never actually took this branch.

Now we can